src/components/home/cobe.jsx

```diff
@@ -0,0 +1,143 @@
+import createGlobe from 'cobe';
+import { useEffect, useRef } from 'react';
+import { useSpring } from 'react-spring';
+
+export default function Cobe() {
+  let phi = 0;
+
+  const canvasRef = useRef(null);
+
+  const pointerInteracting = useRef(null);
+  const pointerInteractionMovement = useRef(0);
+
+  const genrateRandomMarkers = () => {
+    const markers = [];
+
+    for (let i = 0; i < 10000; i++) {
+      markers.push({
+        location: [Math.random() * 180 - 90, Math.random() * 360 - 180],
+        size: Math.random() * 0.1,
+      });
+    }
+
+    return markers;
+  };
+
+  const randomMarkers = genrateRandomMarkers();
+
+  const [{ r }, api] = useSpring(() => ({
+    r: 0,
+    config: {
+      mass: 1,
+      tension: 280,
+      friction: 40,
+      precision: 0.001,
+    },
+  }));
+
+  useEffect(() => {
+    let width = 0;
+
+    const onResize = () => {
+      canvasRef.current && (width = canvasRef.current.offsetWidth);
+    };
+
+    window.addEventListener(`resize`, onResize);
+
+    onResize();
+
+    const globe = createGlobe(canvasRef.current, {
+      devicePixelRatio: 2,
+      width: width * 2,
+      height: width * 2 * 0,
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **question (code_clarification):** The expression 'width * 2 * 0' results in 0; verify if this is intended for the globe's height.